user-docs/latest/content-manager/saving-and-publishing-content.md

```diff
@@ -0,0 +1,46 @@
+# Saving, publishing and deleting content
+
+Strapi allows you to manage your content throughout its whole lifecycle, whether your are working on its draft version, about to finish it and share it with the world, or wanting to delete it when it's obsolete.
+
+## Saving & publishing content
+
+::: warning IMPORTANT
+The possibility to manage drafts for contents comes from the Draft & Publish feature. This feature is activated by default, but it can be disactivated for any content type. If you disabled the Draft & Publish feature, saving your content means saving and publishing at the same time.
+:::
+
+Your contents can have 2 statuses: draft and published. You can see the current status indicated on the right of the interface, below the Information box.
+
+![Editing draft version](../assets/content-manager/editing_draft_version.png)
+
+By default, each newly created content is a draft. Drafts can be modified and saved at will, using the **Save** button on the top right corner of the edit view, until they are ready to be published.
+
+### Publishing a draft
+
+To publish a draft, click on the **Publish** button in the top right corner of the content editor.
```

Comment:
Is it mandatory to mention, that users need to have the specific permission to publish? As an author won't have the permission to do so unlike the editor or the super admin for example

Reply:
In the introduction of the user guide (#8) we mentioned that there is a system of permissions & roles behind most our features, to explain that we don't precise throughout the whole documentation what permissions are needed, but that users should keep in mind that if they can't do something, it may be because they don't have the required permission.